### DISCUSSION TITLE
how to insert `{{xxx}}}`

### DISCUSSION THREAD

**Discussion by wszgrcy** [handlebars-lang/handlebars.js#2013] [Q&A] [answered]
Before filing issues, please check the following points first:

- [x] Please don't open issues for security issues. Instead, file a report at https://www.npmjs.com/advisories/report?package=handlebars
- [x] Have a look at https://github.com/handlebars-lang/handlebars.js/blob/master/CONTRIBUTING.md
- [x] Read the FAQ at https://github.com/handlebars-lang/handlebars.js/blob/master/FAQ.md
- [x] Use the jsfiddle-template at https://jsfiddle.net/4nbwjaqz/4/ to reproduce problems or bugs

This will probably help you to get a solution faster.
For bugs, it would be great to have a PR with a failing test-case.
[https://jsfiddle.net/4qvo810e/](https://jsfiddle.net/4qvo810e/)
I want to insert `{{xxx}}}` and run with `{xxx:1}`,expect result `1}`, but throw error
```
handlebars.js:2992 Uncaught Error: Parse error on line 1:
{{xxx}}}
------^
Expecting 'CLOSE', 'OPEN_SEXPR', 'ID', 'STRING', 'NUMBER', 'BOOLEAN', 'UNDEFINED', 'NULL', 'DATA', got 'CLOSE_UNESCAPED'
    at Parser.parseError (handlebars.js:2992:20)
    at Parser.parse (handlebars.js:3061:31)
    at parseWithoutProcessing (handlebars.js:2710:34)
    at HandlebarsEnvironment.parse (handlebars.js:2716:14)
    at compileInput (handlebars.js:4573:20)
    at ret (handlebars.js:4582:19)
    at ?editor_console=true:119:47
```
I try to add ` ` to solve it .It work,but `space` will effect template.

**Comment by jaylinski** [comment, marked as answer]
You can use the [Whitespace Control](https://handlebarsjs.com/guide/expressions.html#whitespace-control)-character `~`:

```
{{~name~}} }
```

**Reply by wszgrcy** [reply]
thanks .I use `{{name~}}}` and it work.Although not as imagined, it can indeed be used
